heliopy/data/ulysses.py

```diff
@@ -5,10 +5,13 @@
 """
 import os
 import pandas as pd
+import sunpy.timeseries as ts
```

Comment:
I don't think you need this import for this file.

Reply:
Extremely sorry for the unnecessary import!